base/special/rem_pio2.jl

```diff
@@ -123,7 +104,7 @@ end
         end
     end
     y2 = (r-y1)-w
-    return Int(fn), y1, y2
+    return unsafe_trunc(Int, fn), y1, y2
```

Comment:
what happens if it's bigger than typemax(Int), or will this not be called for inputs that big?